static/bidder-info/admatic.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "http://pbs.admatic.com.tr/?host={{.Host}}"

```

Comment:
@bakicam  received 404 response for above endpoint. @bakicam  could you confirm if POST endpoint exist for above endpoint url

```
oh0387@vpn-10-249-108-112 ~ % curl -i --location --request POST 'http://pbs.admatic.com.tr/?host='
HTTP/1.1 404 Not Found
Date: Fri, 05 Apr 2024 11:11:00 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 70
Connection: keep-alive
X-Powered-By: Express
ETag: W/"46-wd7cp2ZuKnXKZ3zACcKiNOD1w60"
working-on: admatic-k8s
CF-Cache-Status: DYNAMIC
...

{"message":"Cannot POST /?host=","error":"Not Found","statusCode":404}%
```

Reply:
@bakicam  any feedback on above comment

Reply:
Hello, we fixed this problem